index.html

```diff
@@ -346,27 +347,48 @@ <h2>About myself</h2>
       </ul>
       <button class="aboutbutton">Get my Resume</button>
         <ul class="skillset">
-          <li>
-            <h3>languages</h3> <img class="arrow" src="./images/down.png" alt="rightarrow" />
+          <li class="languages">
+            <h3>languages</h3> <img class="arrow downnlanguage" src="./images/down.png" alt="rightarrow" />
+            <ul class="languageskills">
+              <li class="lang">
+                <img src="./images/language3.png" alt="js" />
+                <p>Javascript</p>
+              </li>
+              <li class="lang">
+                <img src="./images/language1.png" alt="js" />
+                <p>HTML</p>
+              </li>
+              <li class="lang">
+                <img src="./images/language2.png" alt="js" />
+                <p>CSS</p>
+              </li>
+            </ul>
           </li>
-          <ul class="languageskills">
-            <li class="lang">
-              <img src="./images/language3.png" alt="js" />
-              <p>Javascript</p>
-            </li>
-            <li class="lang">
-              <img src="./images/language1.png" alt="js" />
-              <p>HTML</p>
-            </li>
-            <li class="lang">
-              <img src="./images/language2.png" alt="js" />
-              <p>CSS</p>
-            </li>
-          </ul>
+          
           <li><h3>Frameworks </h3><img class="arrow" src="./images/right.png" alt="rightarrow" /></li>
           <li><h3>Skills </h3><img class="arrow" src="./images/right.png" alt="rightarrow" />
    </li>
         </ul>
     </section>
+    <footer class="contact-form">
+      <h2 class="h-contact">Contact me</h2>
+      <p class="contact-diescription">
+        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+      </p>
+      <form action="#" method="post">
```

Comment:
- [ ] You should use Flexbox to layout the form elements as per the project requirement
![contact-form-elem-flex-layout](https://user-images.githubusercontent.com/44057863/166580964-8303be48-ae3a-4db2-a01a-c1f2554e91b8.png)
.

Reply:
Good job in implementing the changes 👍